### PR TITLE
Multishop import

### LINKED ISSUE #15: Implement skip item for wrong shop
Related to: https://github.com/ongr-io/ConnectionsBundle/issues/88

### PR DESCRIPTION
**Pull request by tfe2012**
Closes #15

### CODE REVIEW

File: Entity/CatalogProductEntityInt.php [hunk 1]
```diff
@@ -59,6 +59,13 @@
     protected $store;
 
     /**
+     * @var EavAttribute
+     *
+     * @ORM\OneToOne(targetEntity="EavAttribute", mappedBy="catalogProductEntityInt")
+     */
+    protected $eavAttribute;
```
Comment (lukasvtechceo) [inline Entity/CatalogProductEntityInt.php:66]: e.a.v? what it stands for? maybe we should make it more meaningfull?

File: Entity/EavAttribute.php [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+namespace ONGR\MagentoConnectorBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Entity for table "eav_attribute".
+ *
+ * @ORM\MappedSuperclass
+ */
+abstract class EavAttribute
```
Comment (lukasvtechceo) [inline Entity/EavAttribute.php:21]: Eav? maybe we should make it more meaningfull?

File: Modifier/CategoryModifier.php [hunk 3]
```diff
@@ -107,7 +107,7 @@ public function addVarcharAttributes($entity, CategoryDocument $document)
         $varcharAttributes = $entity->getVarcharAttributes();
 
         foreach ($varcharAttributes as $attribute) {
-            if ($this->storeId !== $attribute->getStore()) {
+            if ($this->storeId !== $attribute->getStore() && $attribute->getStore() !== 0) {
```
Comment (lukasvtechceo) [inline Modifier/CategoryModifier.php:111]: maybe you want to add comment for this functionality?
as far as I understant its to skip item if its from different shop, but comment would make it more clear. Thats my opinion only :)

File: Modifier/CategoryModifier.php [hunk 4]
```diff
@@ -134,7 +134,7 @@ public function addIntegerAttributes($entity, CategoryDocument $document)
         $integerAttributes = $entity->getIntegerAttributes();
 
         foreach ($integerAttributes as $attribute) {
-            if ($this->storeId !== $attribute->getStore()) {
+            if ($this->storeId !== $attribute->getStore() && $attribute->getStore() !== 0) {
```
Comment (lukasvtechceo) [inline Modifier/CategoryModifier.php:138]: same as above. I would consired adding comment here.

File: Tests/Unit/Entity/CatalogProductEntityIntTest.php [hunk 5]
```diff
@@ -26,6 +26,7 @@ public function getFieldsData()
             ['value'],
             ['product'],
             ['store'],
+            ['eavAttribute'],
```
Comment (lukasvtechceo) [inline Tests/Unit/Entity/CatalogProductEntityIntTest.php:29]: eav

File: Entity/CatalogProductEntityInt.php (hunk 1 above)
Comment (tfe2012) [inline Entity/CatalogProductEntityInt.php:66]: name from magento db

**Pull request merged**